test/Integration/SqlOutputBindingIntegrationTests.cs

```diff
@@ -47,9 +47,6 @@ public void AddProductTest(int id, string name, int cost, SupportedLanguages lan
         [SqlInlineData(1, "Test", 5)]
         [SqlInlineData(0, "", 0)]
         [SqlInlineData(-500, "ABCD", 580)]
-        // Currently PowerShell and Java functions return null when the parameter for name is an empty string
-        // Issue link: https://github.com/Azure/azure-functions-sql-extension/issues/443
-        [UnsupportedLanguages(SupportedLanguages.PowerShell, SupportedLanguages.Java)]
```

Comment:
These are still failing for Java it seems. 

@lucyzhang929 - possibly a similar issue for their worker. So something to follow up with them about when you get around to investigating this.